keyboards/dtd/ldmlKeyboard.dtd

```diff
@@ -34,6 +34,8 @@ Except as contained in this notice, the name of a copyright holder shall not be
 <!ELEMENT names ( name+ ) >
 
 <!ELEMENT name EMPTY >
+<!ATTLIST name type CDATA #IMPLIED > 
+    <!--@MATCH:literal/layout, indicator, normalization--> 
```

Comment:
I think it would be good to have an implied (but legal) `type=` which matches the current use of `<name>`, something like `<name type="display" value="Greek (220) Latin" />`

Reply:
We discussed having permitted normalization types "NFC", "NFD", "mixed" in [CLDR-13273](https://unicode-org.atlassian.net/browse/CLDR-13273). Per Richard's comment, maybe the third type is "other", or possibly omitted is better if the layout is non-conformant. Richard points to the need for compatibility with USE being a third type, ,so "USE", but I don't want to enshrine "USE" this way.